src/main/java/Membership.java

```diff
@@ -22,6 +22,21 @@ public boolean checkMembership(Customer customer) {
         }
         return customer.checkMembership();
     }
+    /**
+     * Check this customer's current membership status
+     *
+     * @param customer The customer needed to Check current membership status.
+     */
+    public int checkMembershiplevel(Customer customer) {
+        if (customer.checkMembership()){
+            if (this.MembershipCustomer.containsKey(customer.getUsername())) {
+                return customer.checkMembershiplevel();}
```

Comment:
lines 33 and 36 and 38 are redundant. May be reduced to one line.

Reply:
for double check purposes, but could remove